ReSharper.FSharp/src/FSharp.Psi.Features/src/Intentions/ToMultilineRecord.fs

```diff
@@ -40,14 +41,15 @@ type ToMultilineRecord(dataProvider: FSharpContextActionDataProvider) =
         let firstBinding = bindings.[0]
         let indent = firstBinding.Indent
 
+        // todo: rewrite using ModificationUtil when code formatter rules are ready
         for binding in bindings do
             if binding != firstBinding then
-                let newLine =
-                    match binding.PrevSibling with
-                    | Whitespace node -> ModificationUtil.ReplaceChild(node, NewLine(lineEnding))
-                    | node -> ModificationUtil.AddChildAfter(node, NewLine(lineEnding))
+                let newLine = NewLine(lineEnding)
+                match binding.PrevSibling with
+                | Whitespace node -> LowLevelModificationUtil.ReplaceChildRange(node, node, newLine)
```

Comment:
Let's use `DisableCodeFormatter` or something similar instead until we're ready to use the formatter here?